src/aioswitcher/api/packets.py

```diff
@@ -20,51 +20,51 @@
 NO_TIMER_REQUESTED = "00000000"
 NON_RECURRING_SCHEDULE = "00"
 # format values are local session id, timestamp
-REQUEST_FORMAT = "{}340001000000000000000000{}00000000000000000000f0fe"
-REQUEST_FORMAT2 = "{}390001000000000000000000{}00000000000000000000f0fe"
+REQUEST_FORMAT_TYPE1 = "{}340001000000000000000000{}00000000000000000000f0fe"
```

Comment:
This still bothers me, we talked about it in the previous PRs, yet we merged it as-is.
Can we do better the TYPE1 vs TYPE2 for all the constants in this file?

If not, maybe consider a dynamic constants script injected by the initiator...

Reply:
I am sorry I am not following you, we had a discussion that we agreed that we split all places that have to TYPE1 & TYPE2 so that it is clear that there are two types of protocols. As far as I see it if you don't agree with this base assumption (I see it repeating in comments). We should close this PR, discuss again and make a new one.

Reply:
I remember discussing it, I remember going along with this.
But I also remember constantly raising my concerns in regards to the naming issues with the new contributions,
and I remember getting constantly reassured that this is a WIP branch and we can address this later.

I'll reiterate my concerns:
to anyone who's not involved with this upcoming release,
stuff like TYPE1/2, FORMAT1/2, and PORT1/2 don't mean much.

I understand the TYPE types and variables are a bit tricky to handle, AFAIK every type represents a set of devices that seemingly doesn't have a specific property that causes them to be in a specific TYPE group.
But I think that at the very least the PORT and PACKET_FORMAT variables can be named better.
Although, I would prefer the TYPEs to have even more meaning as well.

Reply:
Can you suggest something with better meaning? The easiest way would be to review the changes and add code suggestions so I can commit them if they look good.
I don't have any better approach I can think of than what implemented here.

Reply:
> Can you suggest something with better meaning? The easiest way would be to review the changes and add code suggestions so I can commit them if they look good.

So... you don't take reviews without alternative suggestions...
For me to be able to intervene further on a expected level, besides the suggestion I had already made in regards to the port variables naming, which you opposed, means I would have to get more invested in the next release, much like you and Jafar already are, and unfortunaly, I don't think I can spare the time in foreseen future.

You could have specified why is this such an issue and perhaps we could have brainstormed together,
or you could have stated that you've considered everything else but that's the only way and you own it (you are a maintainer here no?), instead, you took the if-you-don't-like-it-then-rewrite-it approach.

Dude, as I told you previously...
you are a maintainer here,
you maintain the main client of this repo (ha),
you are more involved with the users than I am and therefore know what is requested,
you are (partially) connected to the vendor,
you were involved in the planning of these modifications with Jafar.
I don't have the time to catch up on everything **you already know** in regards to this project,
so from my end, **whatever you say goes** - you have my blessing and I would try to help out whichever way I can.

To this, you replied that I can help out with code reviews,
yet it looks like you don't really like my code reviews.
:man_shrugging:

So...
Let's just keep this as-is,
I'll approve and merge this now.